### PR TITLE
ENG-1103: skip-nft-creation-in-dev-environment

### PR DESCRIPTION
**Pull request by stormcloud266**

### CODE REVIEW

**Comment by stormcloud266** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #951** <a href="https://app.graphite.dev/github/pr/infinity-keys/ik-landing/951?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1126** <a href="https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1126?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1127** <a href="https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1127?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
        * **PR #1128** <a href="https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1128?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #1129** <a href="https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1129?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1127?utm_source=stack-comment).

File: api/src/lib/web3/get-nft-data.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+// @NOTE: If an NFT with this token ID exists in the DB, it will throw an error
+const DEFAULT_TOKEN_ID = 100
+
+const getTokenIdFromDB = async () => {
+  const lastTokenRecord = await db.nft.findFirst({
+    orderBy: {
+      tokenId: 'desc',
+    },
+  })
```
Comment (illepic) [inline api/src/lib/web3/get-nft-data.ts:16]: Let's verify that this is actually getting the last. And let's see if Prisma has a `findLast` equivalent.

Comment (illepic) [reply api/src/lib/web3/get-nft-data.ts:16]: Also feel free to throw in more comments about what we're doing here.

**Review by illepic — CHANGES_REQUESTED** [review]

**Review by illepic — APPROVED** [review]

**Comment by stormcloud266** [comment]
### Merge activity

* **Feb 5, 12:05 PM EST**: @stormcloud266 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1127).
* **Feb 5, 12:06 PM EST**: [Graphite](https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1127) rebased this pull request as part of a merge.
* **Feb 5, 12:07 PM EST**: @stormcloud266 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/infinity-keys/ik-landing/1127).

**Pull request merged**